docs/siem/reference/timeline-schema.asciidoc

```diff
@@ -0,0 +1,192 @@
+[[timeline-object-schema]]
+[role="xpack"]
+== Timeline object schema
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|`columns` |<<col-obj, columns[]>> |The timeline columns displayed in the UI.
+|`created` |Float |The time the timeline was created, using a 13-digit Epoch
+timestamp.
+|`createdBy` |String |The user who created the timeline.
+|`dataProviders` |<<dataProvider-obj, dataProviders[]>> |The dropzone query
+that determines which events are displayed in the timeline.???
+|`dateRange` |Object |The timeline's range.
+|`description` |String |The timeline's description.
+|`eventNotes` |<<eventNotes-obj, eventNotes[]>> |Notes added to specific
+events in the timeline.
+|`eventType` |String a|Event types displayed in the timeline, which can be:
+
+* `all`: all events
+* `raw`: raw events only
+* `signal`: signals only
+
+|`favorite` |<<favorite-obj, favorite[]>> |Indicates who marked a timeline as a
+favorite, and at what time.
+|`filters` |<<filters-obj, filters[]>> |Filters used in addition to the
```

Comment:
<img width="791" alt="Screenshot 2020-06-25 at 11 17 08" src="https://user-images.githubusercontent.com/6295984/85701622-7c6c8f00-b6d5-11ea-8e2f-54ca02e44181.png">
<img width="878" alt="Screenshot 2020-06-25 at 11 17 18" src="https://user-images.githubusercontent.com/6295984/85701625-7d052580-b6d5-11ea-8c39-4253d6e8c99e.png">

Reply:
We use this to render the filter under the search box